sklearn/tests/test_common.py

```diff
@@ -134,16 +134,6 @@ def test_configure():
         old_argv = sys.argv
         sys.argv = ['setup.py', 'config']
 
-        # This test will run every setup.py and eventually call
-        # check_openmp_support(), which tries to compile a C file that uses
-        # OpenMP, unless SKLEARN_NO_OPENMP is set. Some users might want to run
-        # the tests without having build-support for OpenMP. In particular, mac
-        # users need to set some environment variables to build with openmp
-        # support, and these might not be set anymore at test time. We thus
```

Comment:
Is this issue (variable not being declared anymore at test time) still an issue here? Maybe not here but somewhere else? (just making sure)

Reply:
No since the build should always succeed now